pkg/sfu/downtrack.go

```diff
@@ -603,16 +625,28 @@ func (d *DownTrack) WriteRTP(extPkt *buffer.ExtPacket, layer int32) error {
 
 	hdr, err := d.getTranslatedRTPHeader(extPkt, tp)
 	if err != nil {
-		d.logger.Errorw("write rtp packet failed", err)
+		d.params.Logger.Errorw("write rtp packet failed", err)
 		if pool != nil {
 			PacketFactory.Put(pool)
 		}
 		return err
 	}
 
+	extensions := []pacer.ExtensionData{{ID: uint8(d.dependencyDescriptorExtID), Payload: tp.ddBytes}}
+	if d.playoutDelayExtID != 0 {
+		if val := d.playoutDelay.Load(); val != nil {
+			b, err := val.(rtpextension.PlayOutDelay).Marshal()
```

Comment:
can you just hold marshalled bytes in the structure and directly use it instead of marshalling every time?